src/main/java/smartthings/ratpack/sqs/DefaultSqsService.java

```diff
@@ -35,7 +37,8 @@ public DefaultSqsService(
     @Override
     public void onStart(StartEvent event) throws Exception {
         if (config.isEnabled()) {
-            this.sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            Optional.ofNullable(config.getEndpoint()).ifPresent(sqs::setEndpoint);
```

Comment:
@jeff-blaisdell 
I researched the existing libraries for integrating ratpack with sqs. I found that this one requires configured aws sqs service which can be unhandy in development stage and in writing integration tests. So this option should be configured as simple as possible. I agree that optional binder is good approach but this is a topic to another pull request.

I choosed spring boot with aws cloud messaging autoconfigurer. But I hope to switch it to this one when it will become ready to production.